### ISSUE TITLE
Gree Pular Pro v 1.23

### ISSUE THREAD

**Issue by karolgoluch** [RobHofmann/HomeAssistant-GreeClimateComponent#194]
Not an actual issue - for anyone wondering how to integrate the Gree Pular Pro with HA - I'd like to share the info that I've successfully set the HACS component to work with the **Gree Pular Pro with firmware 1.23**. 

Steps to reproduce:
-do not enable the built-in Gree climate integration (that was not obvious to me at first)
-my Gree AC is connected to Gree Climate+ cloud, I can control the AC with the original Gree apk and with standard remote.
-install HACS GreeClimateComponent and whats probably most important - set the encryption_version to 2.

Here's my climate.yaml:

`- platform: gree
  name: First AC
  host: <ip of your first AC>
  port: 7000
  mac: '<mac address of your first AC. NOTE: Format can be XX:XX:XX:XX:XX:XX or XX-XX-XX-XX-XX-XX depending on your model>'
  encryption_version: 2
  target_temp_step: 1
  
- platform: gree
  name: Second AC
  host: <ip of your first AC>
  port: 7000
  mac: '<mac address of your first AC. NOTE: Format can be XX:XX:XX:XX:XX:XX or XX-XX-XX-XX-XX-XX depending on your model>'
  encryption_version: 2
  target_temp_step: 1`

AC appeared in the Entities:

![2024-07-16-14-26-25@2x](https://github.com/user-attachments/assets/b0a22e7e-c687-45c8-9841-6c60f4ad19be)

![2024-07-16-14-20-20@2x](https://github.com/user-attachments/assets/4096a444-d8a6-4190-b7a0-87fed978fcec)

And everything works properly.

**Comment by evrenakar** [comment]
Dostum merhaba, gree klimalar sadece 2.4 bant genişliğini kabul ettiğinden onlara özel bir ağ yarattım. HA ile farklı iplerde olduğundan bu şekilde config yaptıgımda HA tarafında görnümüyorlar. IP ve MAC adreslerini nasıl verdin?

**Comment by karolgoluch** [comment]
Both of my AC are in the same network as the HA so I didn't have any issue regarding that.

As example: my HA (raspberry pi) is 192.168.1.150, the AC are 192.168.1.160 and 192.168.1.161.

```
- platform: gree
  name: First AC
  host: 192.168.1.160
  port: 7000
  mac: 'A1-B2-C3-D4-E5-F6'
  encryption_version: 2
  target_temp_step: 1
  
- platform: gree
  name: Second AC
  host: 192.168.1.161
  port: 7000
  mac: 'A1-B2-C3-D4-E5-F6'
  encryption_version: 2
  target_temp_step: 1
```

**Comment by Draghmar** [comment]
Did you make some changes to the way ACs are configured? I can't get it work with this component. :(
Also - how can you have v1.23 when Gree app updates Pular to v2.06? I'm not even sure I had my devices on such low firmware from the very beginning.

**Comment by karolgoluch** [comment]
No, all AC settings are default, the only setup I did is above.
Regarding firmware version - I suppose it depends on the region, I'm in Poland. GREE+ app reports 1.23 as the latest version.

**Comment by Draghmar** [comment]
Well...I'm from Poland as well. :) I've noticed Polish naming that's why I was surprised with firmware difference. Maybe there were revisions along the years. I've got mine only year ago, so...
Anyway, thanks for tips. Will try to tinker with it again. Maybe I'll get lucky. ;)

**Comment by karolgoluch** [comment]
Funny coincidence :) Are you sure yours is Pular **Pro** model?
Pozdrawiam! :)

**Comment by Draghmar** [comment]
Hm...Well, that's explains a lot. I didn't noticed that and I didn't know that there is newer version of Pular. Not sure what's the difference between them, except for Pro being a little bit more powerful, but it looks like there are some changes in communication modules.
